<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="0">

  <testsuite name="logs.test"
    errors="1" failures="0" hostname="" id="" package="test" skipped="0"
    tests="0" time="0.002000" timestamp="">
  

  
    <system-err>
       An error occurred while creating a new Appium Server session: [WebDriverError] An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    </system-err>
  

    
    <error message="An error occurred while creating a new Appium Server session: [WebDriverError] An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up"><![CDATA[
      WebDriverError: An error occurred while creating a new Appium Server session: [WebDriverError] An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
    at Object.throwDecodedError (C:\MobileTesting\node_modules\selenium-webdriver\lib\error.js:524:15)
    at parseHttpResponse (C:\MobileTesting\node_modules\selenium-webdriver\lib\http.js:601:13)
    at Executor.execute (C:\MobileTesting\node_modules\selenium-webdriver\lib\http.js:529:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
      ]]></error>  

  
  </testsuite>
</testsuites>
